### DISCUSSION TITLE
[BUG] training error using deepmd-kit 2.0.0b3

### DISCUSSION THREAD

**Discussion by r-hou** [deepmodeling/deepmd-kit#884] [Q&A]
I encountered this problem while training the model using deepmd-kit 2.0.0b3, I appreciate for any help, thanks!
[](url)
<img width="1167" alt="截屏2021-07-24 下午6 01 19" src="https://user-images.githubusercontent.com/47200139/126865076-52b7a3b8-bb9f-4eb0-a71d-a10aec3b8936.png">

**Comment by njzjz** [comment]
This seems to be a bug. Please provide your input file and data, so we can reproduce the bug.
